dap_prinz_green_jobs/config/base.yaml

```diff
@@ -8,3 +8,22 @@ ojo_s3_file_skills: "s3://open-jobs-lake/latest_output_tables/skills_ojd_daps_ex
 ojo_deduplication_file: "s3://prinz-green-jobs/outputs/data/ojo_application/deduplicated_sample/deduplicated_job_ids.csv"
```

Comment:
side note but given we're using getters to load data instead of names in config, i think we can get rid of `ojo_deduplication_file` and `ojo_sample_file`